solr/core/src/java/org/apache/solr/cloud/api/collections/SplitShardCmd.java

```diff
@@ -80,6 +80,15 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * Credits to Megan Carey for documentation
```

Comment:
(1) the first line/sentence of Javadoc of any class should be about the class, not meta information like this.
(2) at the ASF there is a policy (maybe not strict) against attributing an author of code within the code/docs.  I've found this anti-author policy useful for non open-source as well.  Source control metadata is superior.  And referencing people is a slippery slope of adding more people.  Instead we can reference her in the metadata of the commit message and/or CHANGES.txt if applicable.
  I'm not sure if this is general ASF policy but we don't attribute source code to individuals, and I take this approach to inside companies as well.